### PR TITLE
refactor: #2520 remove unused styles + refactor app revision styles

### PR DESCRIPTION
**Pull request by nphivu414**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2520 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2520 

## What is the new behavior?

- Remove unused styles + refactor app revision styles

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by vuhuucuong — APPROVED** [review]

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx [hunk 1]
```diff
@@ -8,17 +19,17 @@ export interface DiffMediaProps {
...
-  const imageStyle = type === 'icon' ? styles.iconImage : styles.mediaImage
+  const blockStyle = type === 'icon' ? iconBlock : mediaBlock
+  const imageStyle = type === 'icon' ? iconImage : mediaImage
   const isDiff = currentMedia !== changedMedia
   return (
-    <div className={styles.container}>
-      <div className={`${styles.block} ${blockStyle} ${isDiff ? styles.red : ''}`}>
-        <div className={`${styles.image} ${imageStyle}`} style={{ backgroundImage: `url("${currentMedia}")` }} />
+    <div className={container}>
+      <div className={`${block} ${blockStyle} ${isDiff ? red : ''}`}>
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx:27]: Could you change to use cx ?

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx [hunk 2]
```diff
@@ -8,17 +19,17 @@ export interface DiffMediaProps {
...
+        <div className={`${image} ${imageStyle}`} style={{ backgroundImage: `url("${currentMedia}")` }} />
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx:28]: Could you change to use cx ?

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx [hunk 3]
```diff
@@ -8,17 +19,17 @@ export interface DiffMediaProps {
...
       </div>
-      <span className={styles.arrow}>&#8594;</span>
-      <div className={`${styles.block} ${blockStyle} ${isDiff ? styles.green : ''}`}>
-        <div className={`${styles.image} ${imageStyle}`} style={{ backgroundImage: `url("${changedMedia}")` }} />
+      <span className={arrow}>&#8594;</span>
+      <div className={`${block} ${blockStyle} ${isDiff ? green : ''}`}>
+        <div className={`${image} ${imageStyle}`} style={{ backgroundImage: `url("${changedMedia}")` }} />
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-media.tsx:32]: Could you change to use cx ?

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-render-html.tsx [hunk 4]
```diff
@@ -10,10 +10,10 @@ export interface DiffRenderHTMLProps {
 const DiffRenderHTML = ({ currentString, changedString }: DiffRenderHTMLProps) => {
   const isDiff = currentString !== changedString
   return (
-    <div className={styles.container}>
-      <HTMLRender className={`${styles.diffRenderBlock} ${isDiff ? styles.red : ''}`} html={currentString || ''} />
-      <span className={styles.arrow}>&#8594;</span>
-      <HTMLRender className={`${styles.diffRenderBlock} ${isDiff ? styles.green : ''}`} html={changedString || ''} />
+    <div className={diffRenderHtmlContainer}>
+      <HTMLRender className={`${diffRenderBlock} ${isDiff ? red : ''}`} html={currentString || ''} />
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-render-html.tsx:14]: Could you change to use cx ?

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-render-html.tsx [hunk 5]
```diff
@@ -10,10 +10,10 @@ export interface DiffRenderHTMLProps {
...
+      <span className={arrow}>&#8594;</span>
+      <HTMLRender className={`${diffRenderBlock} ${isDiff ? green : ''}`} html={changedString || ''} />
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-render-html.tsx:16]: Could you change to use cx ?

File: packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-viewer.tsx [hunk 6]
```diff
@@ -20,12 +20,12 @@ const diffTypes = {
 const DiffViewer = ({ currentString, changedString, type = 'words' }: DiffViewerProps) => {
   const result = diffTypes[type](currentString, changedString).map((part, index) => {
     return (
-      <span key={index} className={part.added ? styles.green : part.removed ? styles.red : ''}>
+      <span key={index} className={part.added ? green : part.removed ? red : ''}>
```
Comment (tanphamhaiduong) [inline packages/developer-portal/src/components/pages/app-detail/app-revision-modal/app-revision-comparison/diff-viewer.tsx:23]: Could you change to use cx ?

**Review by tanphamhaiduong — APPROVED** [review]

**Pull request merged**